### DISCUSSION TITLE
set rofi to run the menu of selected window/program

### DISCUSSION THREAD

**Discussion by theotrst** [davatorium/rofi#1927] [Q&A]
hi guys,
I know, it is a pretty easy issue, but I am not finding the right command to ask rofi to show the menu (File, Edit, View, Insert...) and submenu  of a selected window/application. 

Thank you
cheers,
T

**Comment by DaveDavenport** [comment]
This is not supported.  There might be scripts available that do this?

I am not even sure if there is a 'generic' way to do this integration.

**Reply by theotrst** [reply]
oh, ok. I had the idea it was possible. thanks anyway.
